Celeste.Mod.mm/Patches/Session.cs

```diff
@@ -1,17 +1,91 @@
 #pragma warning disable CS0626 // Method, operator, or accessor is marked external and has no attributes on it
 
+using Celeste.Mod;
 using Mono.Cecil;
 using Mono.Cecil.Cil;
 using MonoMod;
 using MonoMod.Cil;
 using System;
+using System.Collections.Generic;
+using System.Xml.Serialization;
 
 namespace Celeste {
     public class patch_Session {
+
+        public class Slider {
+            private patch_Session _Session;
+
+            public readonly string Name;
```

Comment:
`public string Name { get; required init; }`? I think it's generally advised to expose properties over readonly fields